### PR TITLE
Remove `entity_from_csv`, tests, and lint

### PR DESCRIPTION
**Pull request by Seth-Rothschild**
Removes `entity_from_csv` as discussed in #57

### CODE REVIEW

File: featuretools/entityset/entityset.py [hunk 1]
```diff
@@ -4,7 +4,6 @@
 from builtins import range, zip
 from collections import defaultdict
 
-import dask.dataframe as dd
```
Comment (kmax12) [inline featuretools/entityset/entityset.py]: can we remove dask as a dependency in featuretools now?

File: featuretools/entityset/entityset.py [hunk 2]
```diff
@@ -299,7 +298,8 @@ def add_column(self, entity_id, column_id, column_data, type=None):
         """
         Add variable to entity's dataframe
         """
-        self.entity_stores[entity_id].add_column(column_id, column_data, type=type)
+        self.entity_stores[entity_id].add_column(
```
Comment (kmax12) [inline featuretools/entityset/entityset.py]: can you undo your line length changes for now? We currently allow lines to go beyond 80 characters in special cases like this one and edits below

File: featuretools/entityset/entityset.py (hunk 1 above)
Comment (Seth-Rothschild) [reply featuretools/entityset/entityset.py]: Looks like it. The only other use is in `load_flight` which gets used in the approximate features docs page (but with a limit on the number of rows). We could just put a hard cap on the number of rows and then not require `dask` projectwide?

**Comment by kmax12** [comment]
looks good to me!

**Pull request merged**